src/calculateRentalCost.js

```diff
@@ -4,7 +4,14 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
+
+  if (days >= 7) {
+    price -= 50;
+  } else if (days >= 3) {
+    price -= 20;
+  }
```

Comment:
You can avoid using `else if` by simply returning within the first `if` block. This simplifies the logic and makes it clearer that the code after the `return` statement will not be executed.